src/config.rs

```diff
@@ -14,235 +14,196 @@ use alloy::{
     /// Key to access Builder transaction submission wallet - AWS Key ID _OR_ local private key.
+    #[from_env(
+        var = "BUILDER_KEY",
+        desc = "Key to access Builder transaction submission wallet - AWS Key ID _OR_ local private key",
+        infallible
+    )]
     pub builder_key: String,
     /// Buffer in seconds in which the `submitBlock` transaction must confirm on the Host chain.
+    #[from_env(
+        var = "BLOCK_CONFIRMATION_BUFFER",
+        desc = "Buffer in seconds in which the `submitBlock` transaction must confirm on the Host chain"
+    )]
     pub block_confirmation_buffer: u64,
-    /// The offset between Unix time and the chain's block times. For Holesky, this is 0; for Ethereum, 11.
-    pub chain_offset: u64,
-    /// The slot time at which the Builder should begin building a block. 0 to begin at the very start of the slot; 6 to begin in the middle; etc.
-    pub target_slot_time: u64,
+
     /// Address on Rollup to which Builder will receive user transaction fees.
+    #[from_env(
+        var = "BUILDER_REWARDS_ADDRESS",
+        desc = "Address on Rollup to which Builder will receive user transaction fees"
+    )]
     pub builder_rewards_address: Address,
     /// Gas limit for RU block.
     /// NOTE: a "smart" builder would determine this programmatically by simulating the block.
+    #[from_env(var = "ROLLUP_BLOCK_GAS_LIMIT", desc = "Gas limit for RU block")]
     pub rollup_block_gas_limit: u64,
     /// URL of the tx pool to poll for incoming transactions.
+    #[from_env(
+        var = "TX_POOL_URL",
+        desc = "URL of the tx pool to poll for incoming transactions",
+        infallible
+    )]
     pub tx_pool_url: Cow<'static, str>,
     /// Duration in seconds transactions can live in the tx-pool cache.
+    #[from_env(
+        var = "AUTH_TOKEN_REFRESH_INTERVAL",
+        desc = "Duration in seconds transactions can live in the tx-pool cache"
+    )]
     pub tx_pool_cache_duration: u64,
     /// OAuth client ID for the builder.
+    #[from_env(var = "TX_POOL_CACHE_DURATION", desc = "OAuth client ID for the builder")]
     pub oauth_client_id: String,
     /// OAuth client secret for the builder.
+    #[from_env(var = "OAUTH_CLIENT_ID", desc = "OAuth client secret for the builder")]
     pub oauth_client_secret: String,
     /// OAuth authenticate URL for the builder for performing OAuth logins.
+    #[from_env(
+        var = "OAUTH_CLIENT_SECRET",
+        desc = "OAuth authenticate URL for the builder for performing OAuth logins"
+    )]
     pub oauth_authenticate_url: String,
     /// OAuth token URL for the builder to get an OAuth2 access token
+    #[from_env(
+        var = "OAUTH_AUTHENTICATE_URL",
+        desc = "OAuth token URL for the builder to get an OAuth2 access token"
+    )]
     pub oauth_token_url: String,
     /// The oauth token refresh interval in seconds.
+    #[from_env(var = "OAUTH_TOKEN_URL", desc = "The oauth token refresh interval in seconds")]
```

Comment:
Should be `OAUTH_TOKEN_REFRESH_INTERVAL`

Reply:
off by one error 😅